google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1185,6 +1187,12 @@ public Builder disableLeaderAwareRouting() {
       return this;
     }
 
+    /** Sets OpenTelemetry object to be used for Spans, Metrics, Views. */
+    public Builder setOpenTelemetry(OpenTelemetry openTelemetry) {
```

Comment:
See also my comment above: This sets a static property. That means that calling this not only affects this `SpannerOptions`, but the entire JVM. If that is intentional, then we should change this also into a static method and document it as such.